Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebButtonsView.swift

```diff
@@ -27,15 +27,15 @@ struct PayPalWebButtonsView: View {
 
                 switch selectedFundingSource {
                 case .paypalCredit:
-                    PayPalCreditButton.Representable(color: .gold, size: .standard) {
+                    PayPalCreditButtonView(color: .gold, edges: .rounded, size: .standard) {
                         payPalWebViewModel.paymentButtonTapped(funding: .paypalCredit)
                     }
                 case .paylater:
-                    PayPalPayLaterButton.Representable(color: .gold, edges: .softEdges, size: .standard) {
+                    PayPalPayLaterButtonView(color: .gold, edges: .rounded, size: .standard) {
                         payPalWebViewModel.paymentButtonTapped(funding: .paylater)
                     }
                 case .paypal:
-                    PayPalButton.Representable(color: .gold, size: .standard) {
+                    PayPalButtonView(color: .gold, size: .standard) {
```

Comment:
Do we want to add `edges: .rounded` here to match the above buttons?

Reply:
I'll add it in